kotlin/tests/pom.xml

```
@@ -117,6 +117,7 @@
           <args>
             <arg>-Werror</arg>
             <arg>-Xuse-experimental=kotlin.ExperimentalStdlibApi</arg>
+              <arg>-XXLanguage:+InlineClasses</arg>
```

```suggestion
              <arg>-Xinline-classes</arg>
```

There's a newer arg for this

---

This fails with
```
[WARNING] Flag is not supported by this version of the compiler: -Xinline-classes
[ERROR] warnings found and -Werror specified
```

---

Interesting, works for me with gradle.